model/model.py

```diff
@@ -133,6 +131,17 @@ def make_random_passenger(self, force: bool=False) -> None:
             p = Passenger(start_floor, end_floor)
             self._passengers.add_passenger(p)
 
+    def make_passengers_from_schedule(self) -> None:
```

Comment:
private so: `_make_passengers_from_schedule`, also can you reorder private methods below public in this class

Reply:
done